core/src/androidMain/kotlin/AndroidAdvertisement.kt

```diff
@@ -1,12 +1,14 @@
 package com.juul.kable
 
+import android.os.Parcelable
+
 @Deprecated(
     message = "Moved as nested class of `AndroidAdvertisement`.",
     replaceWith = ReplaceWith("AndroidAdvertisement.BondState"),
 )
 public typealias BondState = AndroidAdvertisement.BondState
 
-public interface AndroidAdvertisement : Advertisement {
+public interface AndroidAdvertisement : Advertisement, Parcelable {
```

Comment:
This makes this a `major` change, correct?

Reply:
Thank you; yes.